README.md

```diff
@@ -39,99 +39,120 @@ library.
 
 #### Prerequisites
 
-1. The build requires the __cmake__ build system and a compiler toolchain (by default __gcc__).
+1. The build requires Ubuntu 22.04.4 LTS, the __cmake__ build system and a compiler toolchain (by default __gcc__).
```

Comment:
Recommend something along the lines of "BPLib has been tested on Ubuntu 22.04.4 LTS, although it is expected to work on other POSIX-compliant operating systems."